src/Samples/WinAppSdkSample/WinAppSdkSample/WinAppSdkSample.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>

```

Comment:
Do we want this on NET 5 or NET 6 for the purposes of this sample?

Reply:
Both are supported for either should be fine.

Reply:
I'm not sure really, I figure net6 is a better bet?